### ISSUE TITLE
[BUG] image build failed due to missing file at graphite level

### ISSUE THREAD

**Issue by davinkevin** [graphite-project/docker-graphite-statsd#184]
**Please open bug only for Docker component here, if you want to do so for Graphite, please use [graphite-web repo](https://github.com/graphite-project/graphite-web/issues/new/choose)**

**Describe the bug**

The dockerfile provided in this project is not working anymore, failing on the command: 

```
PYTHONPATH=/opt/graphite/webapp /opt/graphite/bin/django-admin.py collectstatic --noinput --settings=graphite.settings
```

**To Reproduce**

Just launch the `build.sh` (without local cache of course): 

```sh
$ ./build.sh
[+] Building 142.9s (22/24)
 => [internal] load build definition from Dockerfile                                                                                 0.0s
 => => transferring dockerfile: 37B                                                                                                  0.0s
 => [internal] load .dockerignore                                                                                                    0.0s
 => => transferring context: 34B                                                                                                     0.0s
 => [internal] load metadata for docker.io/library/alpine:3.14.2                                                                     1.2s
 => CACHED [base 1/3] FROM docker.io/library/alpine:3.14.2@sha256:e1c082e3d3c45cccac829840a25941e679c25d438cc8412c2fa221cf1a824e6a   0.0s
 => => resolve docker.io/library/alpine:3.14.2@sha256:e1c082e3d3c45cccac829840a25941e679c25d438cc8412c2fa221cf1a824e6a               0.0s
 => [internal] load build context                                                                                                    0.0s
 => => transferring context: 3.67kB                                                                                                  0.0s
 => [base 2/3] RUN true  && apk add --update --no-cache       cairo       cairo-dev       findutils       librrd       logrotate     9.1s
 => [base 3/3] RUN apk add --update       collectd collectd-disk collectd-nginx       || true                                        1.0s
 => [build  1/15] RUN true  && apk add --update       alpine-sdk       git       pkgconfig       wget       go       cairo-dev      74.2s
 => [production 1/2] COPY conf /                                                                                                     0.0s
 => [build  2/15] RUN git clone -b 1.1.8 --depth 1 https://github.com/graphite-project/whisper.git /usr/local/src/whisper  && cd /u  1.3s
 => [build  3/15] RUN . /opt/graphite/bin/activate  && git clone -b 1.1.8 --depth 1 https://github.com/graphite-project/carbon.git  11.7s
 => [build  4/15] RUN . /opt/graphite/bin/activate  && git clone -b 1.1.8 --depth 1 https://github.com/graphite-project/graphite-we  8.9s
 => [build  5/15] WORKDIR /opt                                                                                                       0.0s
 => [build  6/15] RUN git clone "https://github.com/statsd/statsd.git"  && cd /opt/statsd  && git checkout tags/v"0.9.0"  && npm i  13.4s
 => [build  7/15] RUN git clone "https://github.com/go-graphite/go-carbon.git" /usr/local/src/go-carbon  && cd /usr/local/src/go-c  13.1s
 => [build  8/15] RUN git clone "https://github.com/lukepalmer/brubeck.git" /usr/local/src/brubeck  && cd /usr/local/src/brubeck &&  8.3s
 => [build  9/15] COPY conf/opt/graphite/conf/                             /opt/defaultconf/graphite/                                0.0s
 => [build 10/15] COPY conf/opt/graphite/webapp/graphite/local_settings.py /opt/defaultconf/graphite/local_settings.py               0.0s
 => [build 11/15] COPY conf/opt/graphite/conf/* /opt/graphite/conf/                                                                  0.0s
 => [build 12/15] COPY conf/opt/graphite/webapp/graphite/local_settings.py /opt/graphite/webapp/graphite/local_settings.py           0.0s
 => [build 13/15] WORKDIR /opt/graphite/webapp                                                                                       0.0s
 => ERROR [build 14/15] RUN mkdir -p /var/log/graphite/   && PYTHONPATH=/opt/graphite/webapp /opt/graphite/bin/django-admin.py coll  0.5s
------
 > [build 14/15] RUN mkdir -p /var/log/graphite/   && PYTHONPATH=/opt/graphite/webapp /opt/graphite/bin/django-admin.py collectstatic --noinput --settings=graphite.settings:
#22 0.420 Traceback (most recent call last):
#22 0.420   File "/opt/graphite/lib/python3.9/site-packages/django/core/management/__init__.py", line 204, in fetch_command
#22 0.420     app_name = commands[subcommand]
#22 0.420 KeyError: 'collectstatic'
#22 0.420
#22 0.420 During handling of the above exception, another exception occurred:
#22 0.420
#22 0.420 Traceback (most recent call last):
#22 0.420   File "/opt/graphite/bin/django-admin.py", line 5, in <module>
#22 0.420     management.execute_from_command_line()
#22 0.420   File "/opt/graphite/lib/python3.9/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
#22 0.421     utility.execute()
#22 0.421   File "/opt/graphite/lib/python3.9/site-packages/django/core/management/__init__.py", line 375, in execute
#22 0.421     self.fetch_command(subcommand).run_from_argv(self.argv)
#22 0.421   File "/opt/graphite/lib/python3.9/site-packages/django/core/management/__init__.py", line 211, in fetch_command
#22 0.421     settings.INSTALLED_APPS
#22 0.421   File "/opt/graphite/lib/python3.9/site-packages/django/conf/__init__.py", line 79, in __getattr__
#22 0.421     self._setup(name)
#22 0.421   File "/opt/graphite/lib/python3.9/site-packages/django/conf/__init__.py", line 66, in _setup
#22 0.422     self._wrapped = Settings(settings_module)
#22 0.422   File "/opt/graphite/lib/python3.9/site-packages/django/conf/__init__.py", line 157, in __init__
#22 0.422     mod = importlib.import_module(self.SETTINGS_MODULE)
#22 0.422   File "/opt/graphite/lib/python3.9/importlib/__init__.py", line 127, in import_module
#22 0.422     return _bootstrap._gcd_import(name[level:], package, level)
#22 0.422   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
#22 0.422   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
#22 0.422   File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
#22 0.422 ModuleNotFoundError: No module named 'graphite.settings'
------
executor failed running [/bin/sh -c mkdir -p /var/log/graphite/   && PYTHONPATH=/opt/graphite/webapp /opt/graphite/bin/django-admin.py collectstatic --noinput --settings=graphite.settings]: exit code: 1
```

**Expected behavior**

A complete build without any errors 

**Environment (please complete the following information):**
 - OS flavor: MacOS, Linux
 - Image version last version from git (1.1.8-2)
 - Custom volumes mounted [e.g. None, /opt/graphite/conf, /opt/graphite/storage etc]: none

**Additional context**

Nothing more, I will dive into this subject and try to find a solution / do a PR, but if you have a better and quicker answer on this one, it's a win-win 😬.

**Comment by deniszh** [comment]
Thanks for report!
Strange though, Django and graphite versions are pinned, maybe some modules
misbehaving.
Will check.

**Comment by davinkevin** [comment]
Strangely, the `pypy` version is still building FYI

**Comment by davinkevin** [comment]
I confirm, this is "ok-ish" with Alpine 3.9 and Python 3.6+. 

The main difference I've seen is after the `graphite-web` installation. With python 3.6.x: 

```
docker run -it --rm --entrypoint=sh docker.io/graphiteapp/graphite-statsd:1.1.8-2 -c 'ls /opt/graphite/webapp/'
content                            graphite_web-1.1.8-py3.6.egg-info
graphite
```

And with Python 3.9.x: 

```
docker run -it --rm --entrypoint=sh graphite-web-3.14.x -c 'ls /opt/graphite/webapp/'
django_tagging-0.4.3-py3.9.egg  graphite_web-1.1.8-py3.9.egg
```

The folder content is totally different with intermediate folders containing version name. I suspect something change at install level if using python 3.7+. I'm not a python expert, so I let you decide the best solution to fix this.

**Comment by davinkevin** [comment]
I think I've found the root cause and provide a PR 🎉

**Comment by deniszh** [comment]
Thanks, fix worked!
I also refreshed Alpine and Django version and released 1.1.8-3
